spec/factories/partners/child.rb

```diff
@@ -9,6 +9,6 @@
     first_name           { Faker::Name.first_name }
     last_name            { Faker::Name.last_name }
     gender               { Faker::Gender.binary_type }
-    item_needed_diaperid { Item.all.sample.id }
+    item_needed_diaperid { Item.all.sample&.id || create(:item).id }
```

Comment:
Similar here - should pass `create(:item, organization: organization)` (I think?)